Source/APIClient/Request.swift

```diff
@@ -18,32 +18,18 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
-    static var authToken: String? { get set }
+    static var authToken: String? { get }
```

Comment:
> If a protocol requires a property to be gettable and settable, that property requirement cannot be fulfilled by a constant stored property or a read-only computed property. If the protocol only requires a property to be gettable, the requirement can be satisfied by any kind of property, and it is valid for the property to be also settable if this is useful for your own code.

Woah, okay so this is counter-intuitive, then, where `{ get set }` is, in a way, _more restrictive_ than simply `get`.

Okay so yeah, this is no problem, then.